__tests__/app.test.js

```diff
@@ -33,40 +33,23 @@ describe("/api/topics", () => {
   });
 });
 
-describe("/api/articles", () => {
-  describe("GET api/articles/:article_id", () => {
-    it("200: it responds with an article object all article properties and author", () => {
+describe("/api/users", () => {
+  describe("GET", () => {
+    it("responds with an array of users", () => {
       return request(app)
-        .get("/api/articles/1")
+        .get("/api/users")
         .expect(200)
         .then(({ body }) => {
-          expect(body.articles).toEqual(
-            expect.objectContaining({
-              article_id: 1,
-              title: "Living in the shadow of a great man",
-              topic: "mitch",
-              author: "jonny",
-              body: "I find this existence challenging",
-              created_at: "2020-07-09T20:11:00.000Z",
-              votes: 100,
-            })
-          );
-        });
-    });
-    it("404: id number not found", () => {
-      return request(app)
-        .get("/api/articles/20")
-        .expect(404)
-        .then(({ body }) => {
-          expect(body.msg).toBe("not found");
-        });
-    });
-    it("400: wrong data type in request", () => {
-      return request(app)
-        .get("/api/articles/one")
-        .expect(400)
-        .then(({ body }) => {
-          expect(body.msg).toBe("bad request");
+          expect(body.users.length > 0).toBe(true);
```

Comment:
Have a look in the test data for how many users there are, then check you are getting back that amount